### DISCUSSION TITLE
Auto instrumentation of metrics to custom open telemetry collector

### DISCUSSION THREAD

**Discussion by padmohan-clgx** [open-telemetry/community#1682] [Community Discussions]
I'm doing auto instrumentation of a python program which is producing message to Kafka. In this we are unable to add CPU usage & Memory usage metrics details for each Span sent to custom open telemetry collector.Please suggest way to send metrics to collector.

**Comment by trask** [comment]
hi @padmohan-clgx! i'd recommend asking this in https://github.com/open-telemetry/opentelemetry-python or [#otel-python](https://cloud-native.slack.com/archives/C01PD4HUVBL)

**Reply by padmohan-clgx** [reply]
Sure, will ask this question in the suggested git hub path. Thank you for the help
